charts/argo-workflows/templates/controller/workflow-role.yaml

```diff
@@ -42,13 +42,15 @@ rules:
       - argoproj.io
     resources:
       - workflowtasksets
+      - workflowartifactgctasks
     verbs:
       - list
       - watch
   - apiGroups:
       - argoproj.io
     resources:
       - workflowtasksets/status
+      - workflowartifactgctasks/status
```

Comment:
Permissions look good :eyes:
https://argoproj.github.io/argo-workflows/walk-through/artifacts/#service-accounts-and-annotations

<img width="816" alt="image" src="https://user-images.githubusercontent.com/30188755/205293878-1987609e-14e4-49b1-833d-bd8374167f12.png">